mod-invoice-storage/examples/document.sample

```diff
@@ -9,7 +9,7 @@
JseqwGApsUWiza8m5KinZKho4qqgWcVVDEKIoEA+HRUawCAAAm0kIzqGIIAt9A6m1pIEL0CACxuO8kCWQ8ThEAVLZQofH3gvHAAdjctjlos2vHuhDDCtQKUxlZz2AeKEErBeXE6gC/NrmzRouFKhc6+ApoAZskGQyw/C8VAQXb2eXE6gC9KrliO7ThGph2/TFUBIgHLAS6XdSooiV2LAocAByBbaIustFRLfpnlBCyKmAx7AzGzoqziavlxcQD4BoU2ATACOgsR0Eu9DjPfLxAPQNImACQAVYERgGQYsIY9Cp4sfWUiAAlgVUACMEN0WsNhT67PVy8QAN+kDPGzQqXa8ztYUex260ATGLD1BiZFQgEYCACdFisp0Z5nGI4tb+l222M2Gzs/cFycDuBb9KFMDRsdYUBZUMTODxiNx8XpAH5URcsRUMKHwrIl2GNmZ4ubBA4BflhJ0wCIAAgjNwHwKAIB8ONq2gZQQlXhmMVBh6s94gHIvHOkRKueqoKTH64WD2Cl5GrxVTJtehoKvWa2TxBWAwA1TzU2atV/xwDQGaJOgQD46W3tA2gsTgAwiwHAdQA/vK8vTgPLzHSyo4RZJoE6gHdwSGkQAY3flQTMwofAOoAM/kMa9tjdxxovxQMIXaV7BOrqtOyvyrzx8pUFEOKNlyoAaKS7RCortewbQ9ZXHgD1GwoEwFtZ5fQAgDlgv02oDuAt/fqMzWlgGXp+/4BAHcB7eOV36E6x2lot+3dk8WUwHQEILYMJAHV0r2B5uZZ9nSy+DL7qAI7bbFk2sHQBYKPnFQiAX28jVdLlquXlWvaVEr+ORygAEwHwb/WTadgzAKZOoTqAX+mj37opDWyrnj+uOB2wFgG1dMO0369lXysyAr4PQE4PALKaAHbzH9KwVwMAv+hQLqcPLRw8qGVfLvOLGQUC4FdepgkAiV/RKRQAu/T0n+kBwMkvaRUPACLgdf5DGvYsAkQBIDvO7/zVb3o9DWyTnj+uQB2gKEYFTPbTF6cOHdKy90sKPa/AIcCvvpbSBAC/llcVAL/mP6RhrwaApiYj3nct+em7gwcOaNn7pSZ6XoEA+P3/aQLAyb9XIBQA+wJEb/oAiPwEQAwArgMiEfbpC/3uX6aB7dZH6HkF6gD+ORzJT98fPnJEy97v5J/vETgE1gG8xX9Iw94viQfAh4Dkp4+wHz6sZe+XEkNAJAD8/k/EmR4AnBF63p8AiAdw8N1f/Le98wlpI4vj+CNZcGPBlpUWt5cSsDUN0dZBE5OYNP5rdxqCFrQq/tm6GFtoo/QQGBjtRaIED4vCEoQ9r1uQeu5JL3pT8FQYLUMOPWRdqSVEKvSwM/n9xk2P67zZfY+dD+jvNvm+j7/35qcH3xDAcB22QMD0dIN+ExovAqYhL8U5AO4CbEjdqOGAGymMS3EOwKsxU4946IBHKbzKk+IWOBeQqEkADFcrBOD1uLwIwOt8bQH0BOBV2Rk+BGTwSm9bgC2A2hxwFcjcvMYBNzMYl+IcYHTAyNi1MYDhOkJ/C2xMTFyd0OBFwATk3aApoL4soH/MiR/EcO3XBdRbIuBF/zPnM4Dh2v/COgGTPAiYtE5AZjLujAMM18mMLYCeAGCjHrhb4+SAmrsYl97/EdpIJq8nNZb46IClJOSluAVsAbYAW8D/W8Dy8vVlDV4ELENeigJkeV7WeMKHgCcy5KU4B6SBOj7mgDqMS3EOkOU0Zx2QptQBtgBbQFnA/Hx2XmN79Om3TwGG6+j2POS1REAVfhDD9VwAvf8rrL0Fsvqxuj30vOo5wHAd2k5DXqoCZjkTMEtJAM4Bs0Cds4oDnHUYl979ApvptFTugMTDqocAwzWhd4BEdQtsZrNSVoMXAVnIawuwBVATMDsrlQ9BPgToh6Cel6oAD2cCPJYIeDzKg4DRxxQFAJse4NYVHuaAK7cwLsU5QJI8ksb0MA8dMDwtQV6KWwAFyFM8CJiSqQv4Yx8ESINx9gXEByUQsG/+srWYCgLypRnPjM44DwLG9aRa3hJevq7GLrr+aAAFfDiYcZcFpEfYFzCS1pNqeQ92UUDgwh3gFUDA++Kqe7XMEvsClspBtbzF9yBA8F5YAPGBAOVPQ0DyEusChpMoYOBYAQG+i6+fPMA36cnvbsBzmfU/Bvzgwai/nWD4ByYEtEEHiHsH2AGr2QW2O+Cn9Cp2wMGeCB3wjQkBARcIUD9/hwI8yV6WBfT+ijnd659VEOAKmBDQ3gkCxHxhMVdmMScPxlkVEB+UMWautpAXQUBnuwkB3lZVxBbYNwTk5IUEmwISC3LOELAPDSC61FYTLwHiiPlQwF4JHjynfUnjTI7EU+MS5NMp7UFwly8WJGb2QI+IBo5/LjOnf8u9TF2Ks/b6S73MGfk0jrF1XT1mdoA2DDZjC7jenZYf3AfPf1v/Y+8QOwKGei8n31bmO32HPzhfc9SUAOLvwLepK1/oq8TtedVwGy9eqPtv6+2GVx73XGW4Qt5I3eE3t34SjfhccJqqa6WdxcW+RYDlulNaU114AkSIWRwhAQcK5aQwMFc7BzBcBwonCg5wQshhWgAJdKMAVTn5uF6LTcZuXf+orR8FdAcIBSKCiA9UD8/YF3B2qBp5hQiN9RNHq/FA7RfjN1tsC9h68x4nQI1WBxUBxB8SDAGicni2tc6qgPWts0NFNAQIIT+d9ZNoe49gCHCpyuti4WindgdgqB4Viq+V85yiYHIE+vog7Gh0/Y2SbymeFo6+Z4ijwmmxJa9UhGzsCNBbPwmG4WV4fhZ82f3l04pOy8oKC/XT2u6Xynxa/4cJVdrbhK8+gPEqtLUTynibfPwI8DV5CXWi+kHAgwB9+0eJFXibOxvZF9DY2ewlVhFuu3+PbQH37reFiYUEw13VdwRWBQh3qrvCQWItQX+sqbq7cixghMbu6qaY3+rlw3HoDwciXaFqhgh1RQJhf5T8iwQdTBEkNjY2Njb/nL8AcRZRWA0Qpu4AAAAASUVORK5CYII="
```

Comment:
missing this part "data:image/png;base64," was confusing the users, so the intent here is to add this part